conjugate_map/conj_calc.py

```diff
@@ -11,10 +12,13 @@
 import numpy as np
 import pandas as pd
 
+logger = logging.getLogger(__name__)
+logging.basicConfig(filename='conjcalc.log', level=logging.DEBUG)
```

Comment:
```suggestion
logging.basicConfig(filename='conjcalc.log', level=logging.INFO)
```

Setting this to DEBUG by default will not be what most users want.  I'd recommend either INFO (as above) or WARNING, since that's what most users care about.